test/e2e/specs/editor/blocks/avatar.spec.js

```diff
@@ -0,0 +1,87 @@
+/**
+ * WordPress dependencies
+ */
+
+const { test, expect } = require( '@wordpress/e2e-test-utils-playwright' );
+
+test.describe( 'Avatar', () => {
+	test.beforeAll( async ( { requestUtils } ) => {
+		await requestUtils.deleteAllUsers();
+	} );
+
+	test.beforeEach( async ( { admin, requestUtils } ) => {
+		await requestUtils.createUser( {
+			username: 'user',
+			email: 'gravatartest@gmail.com',
+			firstName: 'Gravatar',
+			lastName: 'Gravatar',
+			roles: [ 'author' ],
+			password: 'gravatargravatar123magic',
+		} );
+
+		await admin.createNewPost();
+	} );
+
+	test.afterEach( async ( { requestUtils } ) => {
+		await requestUtils.deleteAllUsers();
+	} );
+
+	test( 'should change image when user is changed', async ( {
+		editor,
+		page,
+	} ) => {
+		// Inserting a quote block
+		await editor.insertBlock( {
+			name: 'core/avatar',
+		} );
+
+		const username = 'Gravatar Gravatar';
+
+		const avatarBlock = page.locator(
+			'role=document[name="Block: Avatar"]'
+		);
+
+		await expect( avatarBlock ).toBeVisible();
```

Comment:
This line shouldn't be needed, as there's already `await expect( avatarImage ).toBeVisible();` below, which will assert that the block is visible.